cmd/create/cluster/cmd.go

```diff
@@ -3135,8 +3192,8 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 		}
 	}
 	if spec.ClusterAdminUser != "" {
-		command += fmt.Sprintf(" --cluster-admin-user %s", spec.ClusterAdminUser)
-		command += fmt.Sprintf(" --cluster-admin-password %s", spec.ClusterAdminPassword)
+		command += " --create-admin-user"
```

Comment:
can "--create-admin-password" be used w/o "--create-admin-user", based on above code, seems it's allowed?

Reply:
Yes, it was intentional.
During office hours, I believe the consensus was that we wanted to let the users have three command combinations:
A. --create-admin-user 
> creates the admin user with random password

B. --create-admin-user --create-admin-password
> creates the admin user with user specified password

C. --create-admin-password
> creates the admin user with user specified password

Please correct me if I misunderstood or if I didn't remember correctly. :)

Reply:
right, that's what we want.
But here, if user has selected any option including option C, there will be flag "--create-admin-password". I am not sure it's ok, even I know this will not have impact on the behavior

Reply:
Ah I see what you mean :) Thank you for pointing it out.
Will discuss this during our next call to agree which build command output meets best practice.

Reply:
Moving discussion to: https://github.com/openshift/rosa/pull/1522#discussion_r1342137338